k8s/rabbitmq/README.md

```diff
@@ -339,6 +352,49 @@ for item in service.status.load_balancer.ingress:
 To send and receive messages to RabbitMQ using Python, see the
 [RabbitMQ Tutorial](https://www.rabbitmq.com/tutorials/tutorial-one-python.html).
 
+# Application metrics
+
+## Prometheus metrics
+
+The application is configured to expose its metrics through
+[RabbitMQ Prometheus.io exporter plugin](https://github.com/deadtrickster/prometheus_rabbitmq_exporter) 
+in the [Prometheus format](https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md).
+More details about the plugin setup can be found in the official documentation of RabbitMQ -
+[Monitoring with Prometheus](https://www.rabbitmq.com/prometheus.html)
+Metrics can be read on a single HTTP endpoint available at `[APP_BASE_URL]/api/metrics`,
+where `[APP_BASE_URL]` is the base URL address of the application.
+For example, if you [expose RabbitMQ service internally using port forwarding](#access-rabbitmq-service),
```

Comment:
Please treat it as optional.

I've rephrase this sentence in https://github.com/GoogleCloudPlatform/click-to-deploy/pull/441/files, so this can be something like:

---

```
For example, you can
[expose RabbitMQ service internally using port forwarding](#access-rabbitmq-service),
and then access the metrics by navigating to the [http://localhost:15672/metrics](http://localhost:15672/metrics) endpoint.
```

Reply:
Applied, thanks.